src/core/lv_obj_tree.h

```diff
@@ -123,6 +123,14 @@ void lv_obj_move_up(struct _lv_obj_t* obj);
  */
 void lv_obj_move_down(struct _lv_obj_t* obj);
 
+/**
+ * moves the object to the given index in its parent.
+ * When used in listboxes, it can be used to sort the listbox items.
+ * @param obj  pointer to the object to be moved.
+ * @param index  new index in parent.
```

Comment:
I'd be great to have some examples for specific cases. E.g.
```doxygen
@note to move to the foreground: lv_obj_move_to_index(obj, 0)
@note to move forward (up): lv_obj_move_to_index(obj, lv_obj_get_index(obj) - 1)
```

Reply:
done